tests/end2end-tests/graphstorm-lp/test.sh

```diff
@@ -32,6 +32,11 @@ python3 -m graphstorm.run.gs_link_prediction --workspace $GS_HOME/training_scrip
 
 error_and_exit $?
 
+echo "**************dataset: Movielens, RGCN layer 1, node feat: fixed HF BERT, BERT nodes: movie, inference: mini-batch, negative_sampler: joint, exclude_training_targets: false, eval_batch_size: 1024, no_validation: false"
```

Comment:
Can you just modify 1 train test and 1 inference test instead of adding new tests?

Reply:
You can try to change the first 4 tests in mgpu_test.sh by adding --eval-batch-size 1024

Reply:
Done.